iothub/device/src/Transport/Mqtt/MqttTransportSettings.cs

```diff
@@ -61,7 +61,7 @@ public MqttTransportSettings(TransportType transportType)
             this.QoSPropertyName = "mqtt-qos";
             this.RetainPropertyName = "mqtt-retain";
             this.WillMessage = null;
-            this.DefaultReceiveTimeout = TimeSpan.FromSeconds(DefaultReceiveTimeoutInSeconds);
+            this.SetDefaultReceiveTimeout(TimeSpan.FromSeconds(DefaultReceiveTimeoutInSeconds));
```

Comment:
remove all "this."?

Reply:
I haven't made a full pass on this file, but this one was easy to do, so I did it.